src/action_builddir.rs

```diff
@@ -5,7 +5,7 @@ use crate::tar_check;
 use crate::wrapped;
 use std::path::PathBuf;
 
-pub fn action_builddir(dir: PathBuf, dirs: &RuaDirs, offline: bool, force: bool) {
+pub fn action_builddir(dir: &PathBuf, dirs: &RuaDirs, offline: bool, force: bool) {
```

Comment:
Like `String` and `&str` you should use `PathBuf` and `&Path`.

Reply:
I sometimes forget, unfortunately. Thanks for reminding! Fixed.